### PR TITLE
Adding authorization on Event edit, update, destroy methods

### PR DESCRIPTION
**Pull request by maciek1995**
#73

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,9 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
```
Comment (bpietraga) [inline db/schema.rb]: Duplicated lines

**Comment by bpietraga** [comment]
+1

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -11,7 +12,7 @@ def configure_permitted_parameters
   private
 
   def user_not_authorized
-    flash[:alert] = 'You are not authorized to perform this action.'
-    redirect_to(root_path)
+    flash[:alert] = "You are not authorized to perform this action."
+    redirect_to(request.referrer || root_path)
```
Comment (gotar) [inline app/controllers/application_controller.rb]: in rails you can just do `:back - redirect_to :back`  if I good remember

**Comment by gotar** [comment]
looks good

**Pull request merged**